### ISSUE TITLE
Error: Couldn't fetch pdf for GEOIDs: 06075 from 2000 to 2016

### ISSUE THREAD

**Issue by Lane** [EvictionLab/eviction-maps#1173]
https://sentry.io/eviction-lab/eviction-lab/issues/546268986/

```
Error: Couldn't fetch pdf for GEOIDs: 06075 from 2000 to 2016
  at t.DownloadErrorMessage (/tool/main.7ed3450fa33323f9c3bb.bundle.js:1:1259586)
  at e._error (/tool/main.7ed3450fa33323f9c3bb.bundle.js:1:1259439)
  at e.__tryOrUnsub (/tool/main.7ed3450fa33323f9c3bb.bundle.js:1:33825)
  at e.error (/tool/main.7ed3450fa33323f9c3bb.bundle.js:1:33312)
  at e._error (/tool/main.7ed3450fa33323f9c3bb.bundle.js:1:32229)
...
(17 additional frame(s) were not displayed)
```

**Comment by pjsier** [comment]
So I think I've narrowed this down to two issues. The first is that `serverless-chrome` has some issues with not killing processes in between calls, so sometimes it tries to start up Chrome when it's already running and then fails because Lambda doesn't actually use separate containers for each invocation. The second is that sometimes when this happens, that initial process causing the failure sticks around, so any call after that fails until the Lambda container is recycled.

I'll watch the upstream repo for a stability improvement, but in the meantime I'm trying to mitigate it with the [`refresh-pdf-lambda`](https://github.com/EvictionLab/lambda-utils/tree/master/refresh-pdf-lambda) in the `lambda-utils` repo that updates the description of the PDF Lambda every few hours so that it's guaranteed to recycle the container. From what I'm seeing in Sentry, the PDF error is still happening, but it's less frequent than it was last week before I added that. I'll keep watching the upstream repo for fixes, but in the meantime I think https://github.com/EvictionLab/eviction-lab-exports/issues/68 covers this
